src/main/java/edu/greenblitz/robotName/commands/shooter/pivot/RequirePivot.java

```diff
@@ -0,0 +1,9 @@
+package edu.greenblitz.robotName.commands.shooter.pivot;
+
+public class RequirePivot extends PivotCommand{
```

Comment:
consider changing the name from requirePivot to interrupt pivot. someone might think that this is the way to require a pivot.